pycytominer/tests/test_aggregate.py

```diff
@@ -307,3 +308,32 @@ def test_custom_objectnumber_feature():
     expected_result = expected_result.astype(dtype_convert_dict)
 
     assert aggregate_result.equals(expected_result)
+
+
+def test_output_type():
+    """
+    Testing aggregate pycytominer function
+    """
+    # dictionary with the output name associated with the file type
+    output_dict = {"csv": test_output_file1, "parquet": test_output_file2}
+
+    # test both output types available with output function
+    for _type, outname in output_dict.items():
+        # Test output
+        aggregate(
+            population_df=data_df,
+            strata=["g"],
+            features="infer",
+            operation="median",
+            compute_object_count=True,
+            output_file=outname,
+            output_type=_type,
+        )
+
+    # read files in with pandas
+    csv_df = pd.read_csv(test_output_file1)
+    parquet_df = pd.read_parquet(test_output_file2)
+
+    # check to make sure the files were read in corrrectly as a pd.Dataframe
+    assert type(csv_df) == pd.DataFrame
+    assert type(parquet_df) == pd.DataFrame
```

Comment:
can you also assert their equivalence? 

I think the function is `pd.testing.assert_frame_equal(csc_df, parquet_df)`

Reply:
I have added it now!